src/datapump/sync/sync.py

```diff
@@ -377,67 +402,179 @@ def build_jobs(self, config: DatapumpConfig) -> List[Job]:
         for version in sorted(versions, reverse=True)[:3]:
-            version_prefix = f"{self.SOURCE_PREFIX}{version}"
-            LOGGER.info(f"Looking for RADD tiles in gs://{self.SOURCE_BUCKET}/{version_prefix}")
+            version_prefix = f"{self.source_prefix}{version}"
+            LOGGER.info(
+                f"Looking for RADD tiles in gs://{self.source_bucket}/{version_prefix}"
+            )
 
             version_tiles: int = len(
-                get_gs_files(self.SOURCE_BUCKET, version_prefix, extensions=[".tif"])
+                get_gs_files(self.source_bucket, version_prefix, extensions=[".tif"])
             )
 
-            LOGGER.info(f"Found {version_tiles} RADD tiles in gs://{self.SOURCE_BUCKET}/{version_prefix}")
-            if version_tiles > self.NUMBER_OF_TILES:
+            LOGGER.info(
+                f"Found {version_tiles} RADD tiles in gs://{self.source_bucket}/{version_prefix}"
+            )
+            if version_tiles > self.number_of_tiles:
                 raise Exception(
-                    f"Found {version_tiles} TIFFs in latest RADD GCS folder, which is "
-                    f"greater than the expected {self.NUMBER_OF_TILES}. "
+                    f"Found {version_tiles} TIFFs in latest {self.dataset_name} GCS folder, which is "
+                    f"greater than the expected {self.number_of_tiles}. "
                     "If the extent has grown, update NUMBER_OF_TILES value."
                 )
-            elif version_tiles == self.NUMBER_OF_TILES:
-                return version.rstrip("/")
+            elif version_tiles == self.number_of_tiles:
+                latest_release = version.rstrip("/")
+                source_uris = [
+                    f"gs://{self.source_bucket}/{self.source_prefix}{latest_release}"
+                ]
+
+                return latest_release, source_uris
 
         # We shouldn't get here
-        raise Exception("No complete RADD versions found in GCS!")
+        raise Exception(f"No complete {self.dataset_name} versions found in GCS!")
 
-    @staticmethod
-    def parse_version_as_dt(version: str) -> datetime:
-        # Technically this has a Y10K bug
-        release_date = version.lstrip("v")
-        assert (
-            len(release_date) == 8
-        ), "Possibly malformed version folder name in RADD GCS bucket!"
-        year, month, day = (
-            int(release_date[:4]),
-            int(release_date[4:6]),
-            int(release_date[6:]),
+
+class GLADLAlertsSync(DeforestationAlertsSync):
+    """
+    Defines jobs to create new RADD alerts assets once a new release is available.
+    """
+
+    dataset_name = "umd_glad_landsat_alerts"
+    source_bucket = "earthenginepartners-hansen"
+    source_prefix = "GLADalert/C2"
+    input_calc = "np.ma.array(((A > 0) * (20000 + 10000 * (A > 1) + 2192 + B)) + ((A == 0) * (C > 0) * (20000 + 10000 * (C > 1) + 2192 + D)), mask=False)"
```

Comment:
Hmm, shouldn't the day offset (2192 in both cases) be different for the two sources? Specifically shouldn't it be a year larger for the 2022 data (first prefix)?

Reply:
oh, good catch!